tapir/shifts/forms.py

```diff
@@ -63,6 +63,16 @@ def clean_end_time(self):
 
         return self.cleaned_data["end_time"]
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["num_required_attendances"].initial = 0
```

Comment:
Let's set the default on the model field directly, otherwise we have 2 defaults.

Reply:
okay, I thought we are afraid of too many migrations :laughing:

Reply:
The [doc](https://docs.djangoproject.com/en/5.2/topics/migrations/#squashing-migrations) says we should "not worry about how many you have", suggesting we start looking at it when we have several hundreds, so we're fine for a while.